tests/testing_config/media/Epic Sax Guy 10 Hours.mp4

```diff
@@ -0,0 +1 @@
+I sing a song
```

Comment:
He doesn't actually sing, he is on a sax... 😝 

https://www.youtube.com/watch?v=6iFbuIpe68k